docs/content/annotations.md

```diff
@@ -0,0 +1,189 @@
+
+## Annotations
+
+Name | Type | Description
+--- | --- | ---
+authors | list of strings | A list of authors for the annotation target. Read more [here](./#authors).
+custom | list of arbitrary data | A custom list of named parameters holding arbitrary data. Read more [here](./#custom).
+description | string | A description of the annotation target. Read more [here](./#description).
+organizations | list of strings | A list of organizations related to the annotation target. Read more [here](./#organizations).
+related_resources | list of URLs | A list of URLs pointing to related resources/documentation. Read more [here](./#related-resources).
+schemas | list of object | A list of associations between value paths and schema definitions. Read more [here](./#schemas).
+scope | string; one of `package`, `rule`, `document`, `subpackages` | The scope on which the `schemas` annotation is applied. Read more [here](./#scope).
+
+### Authors
+
+The `authors` annotation is a list of strings, where each entry denotes an author and has the format `{ name } [ "<" email ">"]`; 
+where the name of the author is a sequence of whitespace-separated words. 
+Optionally, the last word may represent an email, if enclosed with `<>`. 
+
+#### Example
+
+```rego
+# METADATA
+# authors:
+# - John Doe
+# ...
+# - Jane Doe <jane@example.com>
+allow {
+  ...
+}
+```
+
+### Custom
+
+The `custom` annotation is a list key value pairs, where the value is arbitrary, user-defined data.
+
+#### Example
+
+```rego
+# METADATA
+# custom:
+# - my_int: 42
+# - my_string: Some text
+# - my_bool: true
+# - my_flag:
+# - my_list:
+#   - a
+#   - b
+# - my_map:
+#    a: 1
+#    b: 2
+allow {
+  ...
+}
+```
+
+### Description
+
+The `description` annotation is a string value describing the annotation target, such as its purpose.
+
+#### Example
+
+```rego
+# METADATA
+# description: The 'allow' rule evaluates to 'true' if ...
+allow {
+  ...
+}
+```
+
+### Organizations
+
+The `organizations` annotation is a list of string values representing the organizations associated with the annotation target.
+
+#### Example
+
+```rego
+# METADATA
+# organizations:
+# - Acme Corp.
+# ...
+# - Tyrell Corp.
+allow {
+  ...
+}
+```
+
+### Related Resources
+
+The `related_resources` annotation is a list of URLs, linking to related resources, such as RFCs and other reading material.
+
+```rego
+# METADATA
+# organizations:
+# - https://example.com/foo
+# ...
+# - https://example.com/bar
+allow {
+  ...
+}
+```
+
+### Schemas
+
+The `schemas` annotation is a list of key value pairs, associating schemas to data values.
+In-depth information on this topic can be found [here](../schemas#schema-annotations).
+
+#### Example
+
+```rego
+# METADATA
+# schemas:
+#   - input: schema.input
+#   - data.acl: schema["acl-schema"]
+allow {
+    access := data.acl["alice"]
+    access[_] == input.operation
+}
+```
+
+### Scope
+
+The `scope` annotation determines how the `schema` annotation will be applied.
```

Comment:
The idea is to change this to make `scope` apply to _all_ annotations. I will do this in a follow-up PR.